matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/AggregatedRelations.kt

```diff
@@ -50,5 +50,6 @@ import com.squareup.moshi.JsonClass
 data class AggregatedRelations(
         @Json(name = "m.annotation") val annotations: AggregatedAnnotation? = null,
         @Json(name = "m.reference") val references: DefaultUnsignedRelationInfo? = null,
+        @Json(name = "m.replace") val replaces: AggregatedReplace? = null,
```

Comment:
I think we may need to update the related Kdoc. And it seems the preview of this doc in the IDE is not working, the `<code></code> `seems to not be well rendered.

Reply:
fixed, should use ```